src/OpenKNX/Console.cpp

```diff
@@ -171,19 +171,32 @@ namespace OpenKNX
             showWatchdogResets(diagnoseKo);
         }
 #endif
-#ifdef ARDUINO_ARCH_RP2040
+#if defined(ARDUINO_ARCH_RP2040) || defined(ARDUINO_ARCH_ESP32)
         else if (!diagnoseKo && (cmd == "fs" || cmd == "files"))
         {
             showFilesystem();
         }
         else if (!diagnoseKo && (cmd == "file dummy"))
         {
+            const char* buffer = "DUMMY";
```

Comment:
dann sollte wir den dateinamen auch in eine variable packen so das beide /dummy.dummy nutzt

Reply:
Wenn wir uns auf ein minimum des API beschränken funktioniert der selbe Code nun für ESP und RP2040.